tachyon/crypto/commitments/batch_commitment_state.h

```diff
@@ -0,0 +1,24 @@
+#ifndef TACHYON_CRYPTO_COMMITMENTS_BATCH_COMMITMENT_STATE_H_
+#define TACHYON_CRYPTO_COMMITMENTS_BATCH_COMMITMENT_STATE_H_
+
+#include <stddef.h>
+
+#include "tachyon/export.h"
+
+namespace tachyon::crypto {
+
+struct TACHYON_EXPORT BatchCommitmentState {
```

Comment:
How about adding constructor of this struct?
That's why, the following code will be simplified.
```c++
  BatchCommitmentState state;
  state.batch_mode = true;
  state.batch_count = num_polys;
```